### DISCUSSION TITLE
Add Ability to Remove Overview Dashboard

### DISCUSSION THREAD

**Discussion by akropp** [home-assistant/frontend#20852] [Dashboard]
While the overview dashboard can be a good start for a fresh installation, in most cases once you have built other dashboards, it is no longer needed or wanted. In addition, with a large installation, the Overview dashboard crashes the mobile client (at least on iOS) because of the number of elements.  Since it is the default whenever you set up a new device or re-login, this make it very difficult to log in.

Can we have the option in the Dashboards page to either delete the Overview dashboard, or at least disable it in a way that it will no longer ever be the default?

**Comment by Nickduino** [comment]
I cannot understand why HA doesn't offer that. They effectively force you to use Overview, so you have to take control of it and edit it to make your own.

**Reply by j4n-e4t** [reply]
While I do agree with that, I think HA doesn't let you remove it because they're working on a better default dashboard that will be auto-generated. However, I would at least want to set a custom default dashboard for every user globally so that they don't always land on "Overview".

**Reply by Nickduino** [reply]
> While I do agree with that, I think HA doesn't let you remove it because they're working on a better default dashboard that will be auto-generated. However, I would at least want to set a custom default dashboard for every user globally so that they don't always land on "Overview".

The appeal of HA is that it allows you to do whatever you want. If I want to remove the Overview dashboard (because I don't want it to be the default dashboard of every new device, for instance), I should be able to do it. And if they create "a better default dashboard that will be auto-generated", I'll create a new dashboard I'll call what I want to check the auto-generated box.

But forcing my hand as if I had bought an Apple device doesn't really feel in line with HA's ethos.

**Reply by akropp** [reply]
That's not really a great reason.  How hard would it be to just have an option in the dashboard page to mark a different dashboard as the default.  Fine, make it the default in absence of another one, but does ANYONE actually keep the overview as their default?  It is totally unusable after you have a couple hundred entities (which is easy to do in any reasonable sized installation).

**Reply by Nickduino** [reply]
> does ANYONE actually keep the overview as their default? It is totally unusable after you have a couple hundred entities (which is easy to do in any reasonable sized installation).

With the way it is implemented, yeah, you're almost forced to keep the overview as your default. **But** you edit it, click the three dots and click "take control", you don't leave the Overview dashboard on "auto", it's unusable.

**Comment by marcinbauer-ohf** [comment]
You can select another dashboard to be the default one by going to Settings / Dashboards and in the edit dialog click "SET AS DEFAULT ON THIS DEVICE". The original "Overview" dashboard will be hidden and remain dormant.

**Reply by Nickduino** [reply]
Yeah but you have to do that for every fucking device.
And it hides overview altogether, so you can't use it as a second dashboard.

**Reply by Kokorev92** [reply]
I confirm! If you reset the cache on the device, you will have to configure it again.

**Reply by akropp** [reply]
And it resets it self randomly every once in a while too.  Not to mention, it really hurts the wife acceptance factor when I can't just say "Install this app and log in" and have it go to the page that actually works.

It wouldn't be such a huge deal if the default overview page didn't render the HA app unusable, such that it is difficult to even make it to the settings page.

**Reply by Ra72xx** [reply]
The same goes for dashboards like Maps, ToDo and Energy, which appear per default on any device and have to be disabled if you don't use them. Moreover, e.g. the Energy dashboard and the ToDo dashboard are not editable, the Map dashboard is only partially editable (no custom icon can be set) and so on.
I really don't get it why Home Assistant - a piece of software I really like btw - forces its ideas of dashboard layout on me...

**Comment by frantathefranta** [comment]
Alternatively, if it's not possible to disable it, could there be a way to fix the issue with it setting itself as default every few weeks? It's frustrating how every single tool dealing with this stops working eventually.
